docs/set-up-replication.md

```diff
@@ -89,11 +78,10 @@ source:
 $ xtrabackup --prepare --target-dir=/path/to/backupdir
 ```
 
-You need to select path where your snapshot has been taken.
+Select path where your snapshot has been taken.
 If everything is ok you should get the same OK message.
```

Comment:
Delete this sentence. Redundant
"If everything is ok you should get the same OK message."

Reply:
removed